ports/esp32/boards/ARDUINO_NANO_ESP32/double_tap.c

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
Could this code be used for other boards (in the future)?  If so maybe it can go in `ports/esp32`?

Reply:
OK, let's keep it here then.